superset/config.py

```diff
@@ -1099,6 +1099,11 @@ def EMAIL_HEADER_MUTATOR(  # pylint: disable=invalid-name,unused-argument
     return msg
 
 
+# Define a list of usernames to be excluded from all dropdown lists of users
+# Owners, filters for created_by, etc.
+EXCLUDE_USER_USERNAMES: List[str] = []
```

Comment:
I assume this is in reference to `User.username`, but wouldn't something like `EXCLUDE_USERS` or `EXCLUDE_USERNAMES` be perfectly understandable, as we're providing the type annotation `List[str]`? Also, should it perhaps mention that it's excluding from lists, so maybe something like `EXCLUDE_USERS_FROM_LISTS` could also work..

Reply:
changed to `EXCLUDE_USERS_FROM_LISTS`